app/(candidate)/dashboard/website/editor/components/PaymentForm.js

```diff
@@ -0,0 +1,120 @@
+'use client'
+
+import { useState } from 'react'
+import {
+  Elements,
+  useStripe,
+  useElements,
+  PaymentElement,
+} from '@stripe/react-stripe-js'
+import { loadStripe } from '@stripe/stripe-js'
+import Button from '@shared/buttons/Button'
+import { useSnackbar } from '@shared/utils/Snackbar'
+import { numberFormatter } from 'helpers/numberHelper'
+
+const stripePromise = loadStripe(process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY)
```

Comment:
Seems like something to wrap into a provider and use-hook since we're doing this in two different places now?